scripts/css/node-vm.ts

```diff
@@ -0,0 +1,48 @@
+export * from 'node:vm';
```

Comment:
Rather than re-exporting everything from the node module, I would rather prefer to create a global `d.ts` declaration for it. Then TypeScript will just use this declaration for the regular `import * as vm from 'node:vm'` statement.